PostBy/CloudCodeSample.js

```diff
@@ -79,3 +79,38 @@ Parse.Cloud.define("deleteAccount", async (request) => {
   // delete account
   await user.destroy({useMasterKey : true});
 });
+
+// Delete Old Posts - Cron Job
+Parse.Cloud.job("removeOldPosts", async (request) => {
+  let date = new Date();
```

Comment:
Is this a constant variable? as well as variables below this line?

Usually we don't expect date/timestamps extracted from system can be altered.

Reply:
It is! I will make sure to change them appropriately.